launch/kinesis-alerts-consumer.yml

```diff
@@ -9,13 +9,12 @@ env:
 - READ_RATE_LIMIT
 - SFX_API_TOKEN
 resources:
-  cpu: 0.2
-  soft_mem_limit: 1.0
-  max_mem: 3.0
+  cpu: 4
```

Comment:
Wow, that's intense. Looking at https://clever.grafana.net/d/3vW3b2cZz/workers?orgId=1&var-environment=production&var-application=kinesis-alerts-consumer&var-pods=All&from=now-7d&to=now, I see the average is much much less, but the max is extremely spiky. We run a large number of these, so I might suggest revisiting the scale after it's working smoothly